apps/website/src/module/root/ui/layout/header/header.presenter.tsx

```diff
@@ -1,29 +1,49 @@
 import type { ComponentPropsWithoutRef, FC } from 'react';
+import { Image } from '@/core/component/image';
 import { Link } from '@/core/component/link';
+import { breakpointTokens } from '@/style/token';
+import MonoDarkLogoImage from '@public/image/logo/mono-dark.png';
+import MonoLightLogoImage from '@public/image/logo/mono-light.png';
 
-type HeaderProps = ComponentPropsWithoutRef<'header'>;
+export type HeaderProps = ComponentPropsWithoutRef<'header'>;
 
-export const Header: FC<HeaderProps> = () => (
-  <header className="sticky top-0 flex w-full flex-col items-center justify-center bg-keyplate-2 p-3">
-    <hgroup className="text-center">
-      <h1 className="my-4 text-4xl font-bold">
-        <span className="rounded-full bg-primary-3 px-8 py-2 text-primary-11">Astronomy Club</span> at&nbsp;NITIC
-      </h1>
-      <p>
-        Next 13 with <code className="bg-keyplate-3 text-keyplate-11">/app</code>{' '}
-        <span className="font-bold text-teal-11">styled using Tailwind CSS</span>
-      </p>
-    </hgroup>
-    <nav className="flex flex-row flex-wrap items-center justify-center gap-4 rounded-full bg-info-3 px-4 py-2">
-      <Link className="text-info-11 underline" href="/article">
-        記事を見る
-      </Link>
-      <Link className="text-info-11 underline" href="/tag">
-        タグ一覧
-      </Link>
-      <Link className="text-info-11 underline" href="/member">
-        部員紹介
+export const Header: FC<HeaderProps> = ({ children, ...props }) => (
+  <header className="sticky top-0 flex w-full flex-col items-center justify-start bg-keyplate-1 font-bold text-keyplate-12" {...props}>
+    <nav aria-label="グローバルナビゲーション" className="flex w-full max-w-md flex-row flex-nowrap items-stretch justify-center gap-6 px-6 py-3">
+      <Link href="/" className="inline-flex shrink-0 items-center duration-100 hover:scale-110 hover:shadow-card">
+        <Image
+          src={MonoLightLogoImage}
+          alt="茨城高専天文部のロゴ"
+          sizes={`20vw, ${breakpointTokens.tablet.mediaQuery} 15vw, ${breakpointTokens.tablet} 10vw`}
+          placeholder="blur"
+          className="w-[50px] dark:hidden"
+        />
+        <Image
+          src={MonoDarkLogoImage}
+          alt="茨城高専天文部のロゴ"
+          sizes={`20vw, ${breakpointTokens.tablet.mediaQuery} 15vw, ${breakpointTokens.tablet} 10vw`}
+          placeholder="blur"
+          className="hidden w-[50px] dark:block"
+        />
       </Link>
+      <ul className="flex shrink grow flex-row items-center justify-between truncate">
+        <li className="inline-flex shrink truncate">
+          <Link href="/article" className="truncate rounded-full px-1.5 py-2 no-underline hover:bg-primary-3 hover:text-primary-11 tablet:px-4">
```

Comment:
`truncate` するなら `title` も指定されてると優しい世界

```suggestion
          <Link title="記事を見る" href="/article" className="truncate rounded-full px-1.5 py-2 no-underline hover:bg-primary-3 hover:text-primary-11 tablet:px-4">
```

Reply:
https://html.spec.whatwg.org/multipage/dom.html#the-title-attribute

設定するなら`aria-label`の方がいいかも？

> Relying on the [title](https://html.spec.whatwg.org/multipage/dom.html#attr-title) attribute is currently discouraged as many user agents do not expose the attribute in an accessible manner as required by this specification (e.g., requiring a pointing device such as a mouse to cause a tooltip to appear, which excludes keyboard-only users and touch-only users, such as anyone with a modern phone or tablet).

A11y的な観点で追加するべきだと提案してくれてたのなら、CSSの`text-overflow`を使って省略するように指定している場合、DOMツリーには全文がレンダーされてるから、そのままでも正しく読み上げてくれるよ
<img width="1756" alt="image" src="https://user-images.githubusercontent.com/16751535/236599270-a9cbfcd1-cf31-4ab2-8f88-12f4a6eb05b9.png">

対応の必要はなさそう？

Reply:
レビューがno contextすぎました :oroka:

目が見える人的には「...」でリンク先が何なのか見えなくなっちゃってるので、`titile` でツールチップ表示させた方がいいよなぁということす
逆に `aria-label` はレス通り DOM にはちゃんと完全系があるので不要っすね

> Relying on the [title](https://html.spec.whatwg.org/multipage/dom.html#attr-title) attribute is currently discouraged as many user agents do not expose the attribute in an accessible manner as required by this specification (e.g., requiring a pointing device such as a mouse to cause a tooltip to appear, which excludes keyboard-only users and touch-only users, such as anyone with a modern phone or tablet).

んーまぁ別に `title` に依存してるわけじゃないし。。使っても良さそう？🤔

https://stackoverflow.com/questions/27953425/what-is-the-difference-between-aria-label-and-title-attributes

Reply:
省略表示が使われるモバイルはポインタデバイスじゃないから追加しても効果的かいまいちわかんないけど、追加しておきました